src/Geometry.h

```diff
@@ -34,9 +34,11 @@ struct fsr_data {
   /** The FSR ID */
   int _fsr_id;
 
-  /** Characteristic point in Universe 0 that lies in FSR */
+  /** Characteristic point in Root Universe that lies in FSR */
   Point* _point;
```

Comment:
Is it necessary to have both a "characteristic point" and a centroid? It seems now that we have the centroid calculator that we can simply use it as the characteristic point (and just use the ``_centroid`` attribute since it is a more intuitive naming convention).